client/extensions/woocommerce/state/reducer.js

```diff
@@ -7,7 +7,9 @@ import { combineReducers } from 'redux';
  * Internal dependencies
  */
 import ui from './ui/reducer';
+import settings from './settings/reducer';
 
 export default combineReducers( {
-	ui
+	ui,
```

Comment:
Hum... I kept the `ui` reducer because it was already there, but for me it made more sense to separate the state tree by feature (`settings`, `products`, `orders`...) instead of `ui` and `api`. The PR that comes after this is based on that assumption (all network-related actions & reducers are part of the `settings/shipping/zones` tree too).

I can see advantages and disadvantages on either approach, I'm willing to discuss it further in Slack.